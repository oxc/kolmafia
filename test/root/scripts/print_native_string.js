const {print} = require("kolmafia");
try {
  print(new String("That's ridiculous. It's not even funny."));
} catch (e) {
  print("exception: " + e.message);
}
