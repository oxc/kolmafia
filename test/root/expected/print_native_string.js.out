That's ridiculous. It's not even funny.
